### ISSUE TITLE
Add node ranking loss function to metrics

### ISSUE THREAD

**Issue by mnarayan** [skggm/skggm#99]
In the context of multi-label regression sklearn has some [metrics]((http://scikit-learn.org/stable/modules/model_evaluation.html#multilabel-ranking-metrics) to assess ranking.

This combined with simple methods to estimate ranks from some underlying continuous attribute can be used to create an additional ranking based loss function to assess estimated inverse covariance or derived networks.
